src/Cryptol/Parser.hs

```diff
@@ -0,0 +1,3995 @@
+{-# OPTIONS_GHC -w #-}
```

Comment:
This file should not be committed to the repo, it is autogenerated from `Parser.y`